src/library/ErrorBoundary/index.tsx

```diff
@@ -14,10 +15,14 @@ export const ErrorFallbackApp = ({
     <h3>
       <FontAwesomeIcon icon={faBug} transform="grow-25" />
     </h3>
-    <h1>Opps, Something Went Wrong</h1>
+    {i18next.resolvedLanguage === 'en' ? (
```

Comment:
ErrrorFallbackApp is still wrapped in I18nextProvider right? So do we need to use resolvedLanguage here? This is also not going to scale as we add more languages.

Reply:
> yes and resolvedLanguage is used for detect the current language for the conditional error content, so i don't see why we don't need it with the I18nextProvider environment?

Can you just use `t` since it is within the context?